tests/vocabulary.rs

```diff
@@ -117,14 +120,34 @@ fn test_real_world() {
                vec![vec![alice, now.clone()], vec![barbara, now.clone()]]);
 }
 
+#[test]
+fn test_default_attributebuilder_complains() {
+    // ::new is helpful. ::default is not.
+    assert!(vocabulary::AttributeBuilder::default()
+                  .value_type(ValueType::String)
+                  .multival(true)
+                  .fulltext(true)
+                  .build()
+                  .validate(|| "Foo".to_string())
+                  .is_err());
+
+    assert!(!vocabulary::AttributeBuilder::new()
```

Comment:
Could this be
```
assert!(vocabulary::AttributeBuilder::new()
    .value_type(ValueType::String)
    ...
    .is_ok());
```

instead? It's much clearer. I missed the negation at the beginning first time I read it.

Reply:
Yes! Excellent idea.